### ISSUE TITLE
[Contacts] Allow contacts to be deactivated and prevent them from logging in to portals

### ISSUE THREAD

**Issue by jstanden** [jstanden/cerb#267]

**Comment by jstanden** [comment]
This should move to identities/portals in 10.0+

**Comment by Hildy** [comment]
With the identity pools, wouldn't this issue become moot?  Just removing a contact/client/identity from the pool should serve this purpose.

**Comment by jstanden** [comment]
@Hildy Yeah, that will be a lot better. We should have never directly used contact records.
